Strikers/Viktor Gyökeres.md

```diff
@@ -1,2 +1,28 @@
 # Viktor Gyökeres
 ## Sporting CP
+
+- Full Name: Viktor Einar Gyökeres
+- Date of birth: Jun 4, 1998 (25)
+- Place of birth: Stockholm, Sweden
+- Height: 1,87 m
+- Citizenship: Sweden, Hungary
+- Position: Attack - Centre-Forward
+- Foot: right
+- Player agent: HCM Sports Management
+- Current club: Sporting CP
+- Joined: Jul 13, 2023
+- Contract expires: Jun 30, 2028
+
+---
+
+- Release clause: €100M
+
+---
+
+## FC 24 Potential
+
+- Rating: 81
+- Potential: 86
+- User feedback: 89
```

Comment:
User feedback must be separate.
Example:

## FC 24 Potential

- Rating: 81
- Potential: 86

---

### User feedback
- Potential: 89
![image]